xml/hwbp_registers.xml

```diff
@@ -980,8 +980,13 @@
     <register name="Instruction Count" short="icount" address="0x7a1">
         This register is accessible as \RcsrTdataOne when \FcsrTdataOneType is 3.
 
-        This trigger matches on every instruction completed or trap taken from a
-        privilege mode where the trigger is enabled.
+        \begin{steps}{This trigger matches at most once per instruction when:}
```

Comment:
Now I'm happy with the bullets but I'm concerned that "at most once per instruction" sounds like it's optional whether it matches at all.  (Maybe an implementation could decide to match zero times per instruction retire in some situations.)  Maybe something like "Unless otherwise disabled/prohibited/specified, this trigger matches when..."

Reply:
What I want to underline is that the trigger matches once even if somehow both of these conditions are met by the same instruction. It doesn't match twice in that case.